common/persistence/nosql/nosqlplugin/cassandra/gocql/client.go

```diff
@@ -24,6 +24,8 @@ import (
 	"crypto/tls"
 	"strings"
 
+	"github.com/uber/cadence/environment"
```

Comment:
can you move this under `"github.com/gocql/gocql"`? 
If you look at any other files, the convention is:
```
 imports of golang standard libary
<newline>
imports of other libraries
<newline>
imports of current repo
```

Reply:
Fixed, funny thing is that `make fmt` wrote it there